src/libraries/System.Threading.RateLimiting/src/System/Threading/RateLimiting/RateLimiterHelper.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace System.Threading.RateLimiting
+{
+    internal static class RateLimiterHelper
+    {
+        internal static readonly double TickFrequency = (double)TimeSpan.TicksPerSecond / Stopwatch.Frequency;
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Is this necessary?

Reply:
![image](https://github.com/user-attachments/assets/2a169ece-2f2b-4938-b5fd-75320ee0ccad)
From the benchmark, it looks better with it
for benchmark code: https://github.com/WeihanLi/PerformanceTest/blob/master/PerformanceTest/StopwatchElapsedTimeTest.cs

Reply:
I imagine the difference here is primarily general CPU noise and not due to inlining vs not, it's on the order of 1-5 instruction cycles.

The actual codegen is likely identical and the attribute isn't needed.

Reply:
Removed